packages/gatsby-theme-aio/src/components/GetCredential/Card/AccessToken.js

```diff
@@ -83,25 +75,9 @@ const AccessToken = ({ accessToken, response, scopesDetails }) => {
                     `}>
                     {credentialToken}
                   </p>
-                  <TooltipTrigger delay={0}>
-                    <ActionButton onPress={() => handleSecretCopyCode(credentialToken)}>
-                      <CopyIcon />
-                    </ActionButton>
-                    <Tooltip>Copy</Tooltip>
-                  </TooltipTrigger>

```

Comment:
Should we keep the copy icon? In case the user wants to copy again?

Reply:
Okay, I'll change that